src/site/xdoc/cmdline.xml.vm

```diff
@@ -450,6 +450,1327 @@ java -D&lt;property&gt;=&lt;value&gt;  \
+ */
+class Test {
+    /**
+     * Demonstrates a fall-through violation in a switch statement.
+     */
+    public void foo() {
+        int i = 0;
+        while (i >= 0) {
+            switch (i) {
+                case 1:
+                case 2:
+                    i++;
+                case 3: // Violation: Fall through from previous branch.
+                    i++;
+            }
+        }
+    }
+}
+  </code></pre>
+</div>
+
+<p><strong>Output:</strong></p>
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-bash">
+Starting audit...
+Skipping excluded file: src/main/java/Test.java
+Audit done.
+  </code></pre>
+</div>
+
+<h4>
+15. Displaying Checkstyle Version (<span class="no-transform">-V</span>, --version)
+</h4>
+<p><strong>Goal:</strong> Print the Checkstyle version information.
+</p>
+
+<p><strong>Command:</strong></p>
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-bash">
+$ java -jar checkstyle-10.21.1-all.jar --version
+  </code></pre>
+</div>
+
+<p><strong>Output (Content of Version Info):</strong></p>
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-bash">
+Checkstyle 10.21.1
+  </code></pre>
+</div>
+
+<h4>
+16. Matching AST Branches (<span class="no-transform">-b</span>, --branch-matching-xpath)
+</h4>
+<p><strong>Goal:</strong>
+Show AST branches matching a given XPath query for the variable declaration of "x".
+</p>
+
+<p><strong>Test File (Test.java):</strong></p>
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-java">
+/**
+ * This is a sample Test class demonstrating Checkstyle AST matching.
+ */
+class Test {
+    /**
+     * Demonstrates extraction of AST nodes for variable definitions.
+     */
+    void foo() {
+        int x = 10;
+    }
+}
+  </code></pre>
+</div>
+
+<p><strong>Command:</strong></p>
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-bash">
+$ java -jar checkstyle-10.21.1-all.jar --branch-matching-xpath "/COMPILATION_UNIT
+/CLASS_DEF/OBJBLOCK/METHOD_DEF/SLIST/VARIABLE_DEF[./IDENT[@text='x']]" Test.java
+  </code></pre>
+</div>
+
+<p><strong>Output (Full AST):</strong></p>
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-bash">
+COMPILATION_UNIT -> COMPILATION_UNIT [4:0]
+`--CLASS_DEF -> CLASS_DEF [4:0]
+    `--OBJBLOCK -> OBJBLOCK [4:11]
+        |--METHOD_DEF -> METHOD_DEF [8:4]
+        |   `--SLIST -> { [8:15]
+        |       |--VARIABLE_DEF -> VARIABLE_DEF [9:8]
+  </code></pre>
+</div>
+
+<h4>
+17. Displaying Help Message (<span class="no-transform">-h</span>, --help)
+</h4>
+<p><strong>Goal:</strong>
+Print usage instructions for the Checkstyle CLI.</p>
+
+<p><strong>Command:</strong></p>
+<div class="wrapper">
+  <pre class="prettyprint"><code class="language-bash">
+$ java -jar checkstyle-10.21.1-all.jar --help
+  </code></pre>
+</div>
+
+<p><strong>Output (Content of Help):</strong></p>
```

Comment:
remove ` (Content of Help)`

what ever is printed in standard output should be just "Output"